File: algoritmos/03 - Estrutura condicional/ec_02/Questões/questao04.py
# Questão 04

nota1 = float(input("Escreva a 1ª nota: "))
nota2 = float(input("Escreva a 2ª nota: "))
nota3 = float(input("Escreva a 3ª nota: "))
nota4 = float(input("Escreva a 4ª nota: "))

notaExm = float(input("Escreva a nota do exame: "))

# Média 01
media = (nota1+nota2+nota3+nota4) / 4
resultado = ""
newMedia = (notaExm + media) / 2
mediaFinal = ""

if media >= 7.0:
    resultado = "Aluno aprovado."
elif media >= 5.0 and media <= 6.9:
    resultado = "Aluno em exame."
else:
    resultado = "Aluno reprovado."

# Média 02
if newMedia >= 5.0:
    mediaFinal = "Aluno aprovado."
else:
    mediaFinal = "Aluno reprovado."

print(f"Média: {media:.1f}")
print(resultado)
print(f"Nota do exame: {notaExm:.1f}")
print(mediaFinal)
print(f"Média final: {newMedia:.1f}")

File: estrutura de dados/2 - Condicional e Repetição/questoes/questao04.c
#include <stdio.h>

int main() {
    float n1, n2, n3, n4, media;

    printf("Digite as quatro notas do aluno:\n");
    scanf("%f %f %f %f", &n1, &n2, &n3, &n4);

    media = (n1 + n2 + n3 + n4) / 4;

    printf("Média: %.1f\n", media);
    if (media >= 7.0) {
        printf("Aluno aprovado.\n");
    } else if (media < 5.0) {
        printf("Aluno reprovado.\n");
    } else {
        printf("Aluno em exame.\n");
    }

    return 0;
}
